### DISCUSSION TITLE
Problems when setting customBreakpoints

### DISCUSSION THREAD

**Discussion by Viraj24Gupta** [twilio-labs/paste#3937] [General] [answered]
Hi there,
I have used `customBreakpoints` using the `CustomizationProvider` and am facing 2 problems as of now.

1. When i try to use SidePanel it covers the whole page width instead of just keeping the width fixed as it does normally.
2. When i pass this `display={['none', 'flex', 'flex']}` to a box component it doesn't work and just uses the first value at every breakpoint.

**Comment by nkrantz** [comment]
Hey @Viraj24Gupta - could you share a reproduction or link to your code so I can take a look?

The Side Panel component is set to take up the full width of the page at our smallest breakpoint, so that's likely why you're seeing that behavior. I'll take a look at your code but if you're trying to keep a fixed width at your smallest breakpoint, you'll probably just have to target that component directly within the Customization Provider.

**Reply by Viraj24Gupta** [reply]
hey sorry for the confusion,
what i meant was even if I am on a bigger breakpoint, say 1024px the side panel is coming out from the right and covering the whole space of the body.

also the arrays that we can use to set values for different styling properties for any component does not work at all.


below is the code snippet I have used to set the custom breakpoints in my app,

```
<CustomizationProvider baseTheme="default" customBreakpoints={["450", "768", "1024"]}>
    <Box display={["none", "flex", "flex"]}> // refering it as box1 below
           some code
     </Box>
</CustomizationProvider>
``` 

box1 is not working as it is supposed to i.e., for width less than 450px it should not be displayed and above that it should be,
but it is not gettinh displayed at any width.

**Reply by krisantrobus** [reply, marked as answer]
Hey @Viraj24Gupta I took a look into this and managed to get it working in this [CodeSandbox](https://codesandbox.io/p/sandbox/customization-check-skh27c?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clxd8qctd00063b6o4jxds14y%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clxd8qctd00023b6o9srsy09k%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clxd8qctd00033b6ospfka0q4%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clxd8qctd00053b6okc1ca3d0%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B65.75662567070873%252C34.24337432929127%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clxd8qctd00023b6o9srsy09k%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clxd8qctc00013b6oiwdvcxsr%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.tsx%2522%257D%255D%252C%2522id%2522%253A%2522clxd8qctd00023b6o9srsy09k%2522%252C%2522activeTabId%2522%253A%2522clxd8qctc00013b6oiwdvcxsr%2522%257D%252C%2522clxd8qctd00053b6okc1ca3d0%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clxd8qctd00043b6oyyb3xteu%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clxd8qctd00053b6okc1ca3d0%2522%252C%2522activeTabId%2522%253A%2522clxd8qctd00043b6oyyb3xteu%2522%257D%252C%2522clxd8qctd00033b6ospfka0q4%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clxd8qctd00033b6ospfka0q4%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D). You are missing the "px" in the breakpoints array. Let me know if this resolves your issue.
```
<CustomizationProvider baseTheme="default" customBreakpoints={["450px", "768px", "1024px"]}>
    <Box display={["none", "flex", "flex"]}> // refering it as box1 below
           some code
     </Box>
</CustomizationProvider>
```

**Reply by roxanagomez** [reply]
Hi @Viraj24Gupta let us know if the solution above solved your issue or if you need further help.

**Reply by Viraj24Gupta** [reply]
hey, yea this resolved my issue
